### DISCUSSION TITLE
Patrol test to support testing the Flutter in-app-web-view

### DISCUSSION THREAD

**Discussion by iam-kj** [leancodepl/patrol#1261] [Q&A]
Currently am looking for automating the Flutter in-app-web-view, is it possible to get this automated using patrol? if so, can we integrate Patrol methods with Flutter Integration test widgets?

I have flutter screen with trading view charts sdk integrated on it, which is completely in_app_web_view & not possible to get this interacted by Flutter Test right now.!

@bartekpacia is this possible to get it done via Patrol??

**Comment by bartekpacia** [comment]
Hi @iam-kj,

In general: yes, Patrol lets you interact with native WebViews in your apps on iOS and Android.

There are a few issues that we haven't yet had time to fix (but we definitely _will_ fix them... soon).


- For Android, it [doesn't work by default](https://github.com/leancodepl/patrol/issues/244#issuecomment-1542164972) (duh!), but [there's a workaround] (https://github.com/leancodepl/patrol/issues/244#issuecomment-1542164972)
- https://github.com/leancodepl/patrol/issues/663
- https://github.com/leancodepl/patrol/issues/1139

**Comment by shubham-jain-tide** [comment]
If i want to access the web component , so i can do that. ??
